src/config.rs

```diff
@@ -902,10 +907,23 @@ impl Config {
             .map(|p| p.as_path())
     }
 
-    pub fn shell(&self) -> Option<&Path> {
-        self.shell
-            .as_ref()
-            .map(PathBuf::as_path)
+    pub fn args(&self) -> Option<Vec<OsString>> {
+        if let Some(ref args) = self.args {
```

Comment:
You can `args.iter().map(|arg| arg.into()).collect<Vec<OsString>>` or something like that here